src/application/validations.ts

```diff
@@ -233,12 +235,31 @@ export const validateApplicationForm =
     const searchSingleSection = (
       formSection: FormSection,
       section: ApplicationFormNode,
-      path: string
+      path: string,
     ) => {
       if (!section) {
         return;
       }
 
+      let optionalFieldsCheckbox: ApplicationField | undefined;
```

Comment:
I would recommend putting this piece of logic in a function, and then name the function so that it is apparent what is being done